openmdao/core/system.py

```diff
@@ -886,6 +1494,23 @@ def _setup(self, comm, mode, prob_meta):
 
         self._setup_var_data()
 
+        # have to do this again because we are passed the point in
+        # # _setup_var_data when this happens
+        self._has_output_scaling = False
```

Comment:
Do you think we should remove the code from the place where we do this the first time?

Reply:
Unfortunately, that does not work. I tried it and ~100 of the tests failed, with errors of this form

```  File "/Users/hschilli/Documents/OpenMDAO/dev/I2650_modify_bounds_scaling_POEM_072/openmdao/test_suite/test_examples/test_circuit_analysis_derivs.py", line 137, in test_nonlinear_circuit_analysis
    p.run_model()
  File "/Users/hschilli/Documents/OpenMDAO/dev/I2650_modify_bounds_scaling_POEM_072/openmdao/core/problem.py", line 842, in run_model
    self.final_setup()
  File "/Users/hschilli/Documents/OpenMDAO/dev/I2650_modify_bounds_scaling_POEM_072/openmdao/core/problem.py", line 1203, in final_setup
    self.model._final_setup(self.comm)
  File "/Users/hschilli/Documents/OpenMDAO/dev/I2650_modify_bounds_scaling_POEM_072/openmdao/core/system.py", line 1607, in _final_setup
    self._setup_vectors(self._get_root_vectors())
  File "/Users/hschilli/Documents/OpenMDAO/dev/I2650_modify_bounds_scaling_POEM_072/openmdao/core/system.py", line 2509, in _setup_vectors
    subsys._setup_vectors(root_vectors)
  File "/Users/hschilli/Documents/OpenMDAO/dev/I2650_modify_bounds_scaling_POEM_072/openmdao/core/system.py", line 2509, in _setup_vectors
    subsys._setup_vectors(root_vectors)
  File "/Users/hschilli/Documents/OpenMDAO/dev/I2650_modify_bounds_scaling_POEM_072/openmdao/core/implicitcomponent.py", line 487, in _setup_vectors
    super()._setup_vectors(root_vectors)
  File "/Users/hschilli/Documents/OpenMDAO/dev/I2650_modify_bounds_scaling_POEM_072/openmdao/core/system.py", line 2492, in _setup_vectors
    vectors[kind][vec_name] = vector_class(
  File "/Users/hschilli/Documents/OpenMDAO/dev/I2650_modify_bounds_scaling_POEM_072/openmdao/vectors/vector.py", line 148, in __init__
    self._initialize_data(root_vector)
  File "/Users/hschilli/Documents/OpenMDAO/dev/I2650_modify_bounds_scaling_POEM_072/openmdao/vectors/default_vector.py", line 132, in _initialize_data
    self._data, self._scaling = self._extract_root_data()
  File "/Users/hschilli/Documents/OpenMDAO/dev/I2650_modify_bounds_scaling_POEM_072/openmdao/vectors/default_vector.py", line 92, in _extract_root_data
    rs0 = root_scale[0]
TypeError: 'NoneType' object is not subscriptable```